### PR TITLE
Norwegian translation (#2492)

### PR DESCRIPTION
**Pull request by sethherr**
Continue #2492 on a Bike Index owned branch

### CODE REVIEW

File: spec/requests/locale_detection_request_spec.rb [hunk 1]
```diff
@@ -40,6 +44,8 @@
         expect(response.body).to match(/bike registration/i)
         get "/", params: {locale: :nl}
         expect(response.body).to match(/fietsregistratie/i)
+        get "/", params: {locale: :nb}
+        expect(response.body).to match(/NORWEGIAN - replace me/i)
```
Comment (sethherr) [inline spec/requests/locale_detection_request_spec.rb:48]: @eld120 this should also be actual norwegian.

File: spec/requests/locale_detection_request_spec.rb [hunk 2]
```diff
@@ -31,6 +31,10 @@
         current_user.update(preferred_language: :nl)
         get "/"
         expect(response.body).to match(/fietsregistratie/i)
+
+        current_user.update(preferred_language: :nb)
+        get "/"
+        expect(response.body).to match(/NORWEGIAN - replace me/i)
```
[the remark this answers is not in the document]
Comment (eld120) [reply spec/requests/locale_detection_request_spec.rb]: @sethherr updated

**Comment by eld120** [comment]
@sethherr We seem to have lost our Norwegian translation on translation.io, meaning it's currently not an option for translation on the platform. This branch is configured correctly, ie `config.target_locales = %w[nl nb]` so you may have to run `rake translation:sync` again

**Comment by sethherr** [comment]
Ahhh, thanks for calling that out @eld120 - every time that we merge to main, it's going to delete the norwegian translation.

I've re-synced it, and I will try not to delete it again.

Let me think about solutions to this.

File: config/locales/translation.nb.yml [hunk 3]
```diff
@@ -3347,14 +3383,14 @@ nb:
...
     currencies:
-      en:
-      en-GB:
-      es:
-      nb:
-      nl:
+      en: USD
+      en-GB: GBP
+      es: EUR
+      nb: kr
```
Comment (eld120) [inline config/locales/translation.nb.yml]: I seem to have missed this line, I just updated it in translation io from 'kr' to 'EUR'. sorry seth

**Pull request merged**